style.css

```
@@ -0,0 +1,101 @@
+
+body {
+  font-family: "Poppins", sans-serif;
+  padding: 0%;
+  margin: 0;
+}
+
+header {
+  height: auto;
+  width: 100%;
+  border-radius: 0.25rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+}
+
+.my-logo {
+  width: fit-content;
+  font-style: normal;
+  color: #6070ff;
+  font-size: 18px;
+  line-height: 20px;
+  text-align: center;
+  font-weight: 700;
+  margin: 10px 12px;
+  text-decoration: none;
+}
+
+.nav-bar {
+  width: 18px;
+  height: 12px;
+}
+
+.menu-buttom {
+  margin: 6px 16px;
+  text-align: center;
+}
+
+section {
+  background: url(images/header-shapes\ mobile@2x.svg);
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+  padding: 0 20px;
+}
+
+.my-name {
+  width: 327;
+  height: 150;
+  font-family: "Poppins", sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  margin-bottom: 10px;
+  letter-spacing: 0.37px;
+  padding-top: 100px;
+  color: #091e42;
+}
```

- [ ] As the previous reviewer suggested, please consider using the right colors for your text. One of the main goals of this activity is that you showcase your skills to follow as close as possible a given design, so please consider making this change! 🤓 🔧 

```suggestion
.my-name {
  width: 327;
  height: 150;
  font-family: "Poppins", sans-serif;
  font-size: 40px;
  font-weight: 700;
  line-height: 52px;
  margin-bottom: 10px;
  letter-spacing: 0.37px;
  padding-top: 100px;
  color: #172b4d;
}
```